__tests__/app.test.js

```diff
@@ -46,7 +60,7 @@ describe("GET /api/article/:article_id", () => {
   });
 });
```

Comment:
How about a 400: article_id = 'cheese'?

Reply:
is my article_id = hello not cheesy enough? :)

Reply:
OK, you win!